src/docserv/docserv.py

```diff
@@ -325,6 +328,43 @@ def start(self):
         Create worker and REST API threads.
         """
         try:
+            # After starting docserv, make sure to stitch as the first thing,
+            # this increases startup time but means that as long as the config
+            # does not change, we don't have to do another complete validation
+            for target in self.config['targets']:
+                self.stitch_tmp_dir = os.path.join(tempfile.gettempdir(), 'docserv_stitch')
+                try:
+                    os.mkdir(self.stitch_tmp_dir)
+                except FileExistsError:
+                    pass
```

Comment:
Use `contextlib.suppress` for lines 336-339 (as shown above).

Reply:
I've replaced that usage with os.makedirs() already, so the try/catch is not necessary anymore.